scripts/exregional_run_prepstart.sh

```diff
@@ -882,6 +882,10 @@ fi
 # 
 if [ ${SFC_CYC} -eq 3 ] ; then
 
+   do_lake_surgery=".false."
+   if [ "${USE_CLM}" = "TRUE" ]; then
```

Comment:
@hu5970 MIng, where does USE_CLM come from?

Reply:
This USE_CLM can be set in configuration step. It is now in var_defin file. The default is FALSE, not using the CLM. We can turn it on when generates the workflow.

Reply:
Please check "ush/config_defaults.sh" change for this variable.

Reply:
@hu5970 I see, thank you for explaining.